src/auth/index.js

```diff
@@ -118,6 +118,41 @@ export default class Auth {
       .then(responseHandler);
   }
 
+  /**
+   * Exchanges a code obtained via a native social authentication solution for the user's tokens
+   *
+   * @param {Object} parameters parameters used to obtain tokens from a code
+   * @param {String} parameters.subjectToken code returned by the native social authentication solution
+   * @param {String} parameters.subjectTokenType identifier that indicates the type of `subjectToken`
+   * @param {String} parameters.userProfile optional element used for native iOS interactions for which profile updates can occur. Only for `APPLE-AUTHZ-CODE`
```

Comment:
optionality is determined by the use of brackets when describing a parameter

```suggestion
   * @param {String} [parameters.userProfile] additional profile attributes to set or override, only on select native social authentication solutions
```

Reply:
The return type is wrong also, will update it. It should't be an array, but an object.

Reply:
> optionality is determined by the use of brackets when describing a parameter

The square brackets are used in typescript definitions to mark a given parameter as optional. So instead of saying "optional parameter to..." you put the brackets and that's it.

Regarding types, those are explained by the curly braces. So for this line, it's saying that is `an optional String property named userProfile`.

Use the above to fix the remaining comments (the ones below this line)

Reply:
Fixed in b6cf3cb8110a349ce96ca14b5512c3e17a42fa57.